static/js/lightbox.js

```diff
@@ -89,13 +90,8 @@ exports.open = function (image, options) {
 
     const $image = $(image);
 
-    // if the asset_map already contains the metadata required to display the
-    // asset, just recall that metadata.
     const $preview_src = $image.attr("src");
     const payload = asset_map.get($preview_src);
-    if (payload === undefined) {
-        payload = exports.parse_image_data($image)
-    }
```

Comment:
I think removing this breaks the `show_from_selected_message` used by the `v` hotkey, since that calls the lightbox without parsing it explicitly.

I think the right version of this commit should just add a parse call in `render_lightbox_list_images`, without the other changes.  It's only in rendering that list that we generate new HTML objects that could not be parsed normally.